<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RTPTechGroup::ModelerIde::PropNumerator</class>
 <widget class="QWidget" name="RTPTechGroup::ModelerIde::PropNumerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPropTool">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButtonAddNumerator">
       <property name="text">
        <string>Добавить</string>
       </property>
       <property name="icon">
        <iconset resource="modeleride.qrc">
         <normaloff>:/add</normaloff>:/add</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonEditNumerator">
       <property name="text">
        <string>Изменить</string>
       </property>
       <property name="icon">
        <iconset resource="modeleride.qrc">
         <normaloff>:/edit</normaloff>:/edit</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonDelNumerator">
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="icon">
        <iconset resource="modeleride.qrc">
         <normaloff>:/remove</normaloff>:/remove</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPropSave">
     <item>
      <spacer name="horizontalSpacerRightProp">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPropSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPropCancel">
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerLeftProp">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="modeleride.qrc"/>
 </resources>
 <connections/>
</ui>
